fetch.bs

```diff
@@ -4948,9 +5039,20 @@ constructor must run these steps:
    to <var>method</var>.
   </ol>
 
- <li><p>Let <var>r</var> be a new {{Request}} object associated with <var>request</var> and
+ <li><p>If <var>init</var>'s <code>signal</code> member is present, then set <var>signal</var> to
+ it.
+
+ <li><p>Let <var>r</var> be a new {{Request}} object associated with <var>request</var>,
  a new associated {{Headers}} object whose <a for=Headers>guard</a>
- is "<code>request</code>".
+ is "<code>request</code>", and a new {{AbortSignal}} object.
```

Comment:
This sentence as exists is confusing, and you made it a bit worse. A band-aid is to maybe add "associated", but a proper fix would be to set _r_'s [=header=] and [=signal=] fields in explicit steps.